FreeRTOS/FreeRTOS-Partner-Supported-Demos
# Update Xtensa FreeRTOS support to v11

Bring Xtensa demos up to date with FreeRTOS version 11, moving to Partner-Supported-Demos submodules.

Description
-----------
Bring existing Cadence-maintained Xtensa v10.4.4-stable demos up to date in the FreeRTOS v11 context:

- Incorporate and consolidate all existing Xtensa test/example bug-fixes from internal, external, and third-party repositories
- Include Xtensa stress test as detailed in Demo/ThirdParty/Template/README.md
- Update FreeRTOSConfig.h for use with all examples.
- Maintain consistent formatting, copyright, and whitespace

Test Steps
-----------
- Using included Makefile and a newer RJ.x Xtensa toolchain for your Xtensa configuration, build the examples and stress test from the command line
- Run on Xtensa instruction-set simulator (ISS) using xt-run or the included Makefile targets

Related Issue
-----------
N/A


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
